cmd/gb-vendor/vendor/manifest.go

```diff
@@ -137,3 +141,14 @@ func readManifest(r io.Reader) (*Manifest, error) {
 	err := d.Decode(&m)
 	return &m, err
 }
+
+// Implement sort for dependencies
+type sortableDependencies []Dependency
```

Comment:
please call this byImportpath (or ByImportpath) so you get 

     sort.Sort(byImportpath(m.Dependencies))